src/main/java/core/basesyntax/service/impl/WriterServiceImpl.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.WriterService;
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.StandardOpenOption;
+
+public class WriterServiceImpl implements WriterService {
+    @Override
+    public void writeDataToFile(String data, String fileName) {
+        File report = new File(fileName);
+        try {
+            report.createNewFile();
+            Files.write(report.toPath(), data.getBytes(), StandardOpenOption.APPEND);
```

Comment:
what does the last parameter mean? do we need this?

Reply:
yes, we need. every time method was called it must append new info to file